pypeman/commands.py

```diff
@@ -108,15 +114,66 @@ def main(debug_asyncio=False, profile=False, cli=False):
     loop.close()
 
 
+def mk_daemon(mainfunc, pidfile="pypeman.pid"):
+    # TODO: might move to a separate module like e.g.  pypeman.helpers.daemon
+    # might also look at following alternative modules:
+    # - python-daemon
+    # - daemonocle
+    # - py daemoniker
+    # Alternatively if we don't want other module dependencies we might just copy
+    # the DaemonLite files into your source repository 
+    if DaemonLite is None:
+        class FakeDaemonizedApp:
+            def __init__(self, *args, **kwargs):
+                pass
+            def start(self):
+                print("running without daemon mode")
+                mainfunc()
+            def stop(self):
+                print("can't stop daemon without module DaemonLite")
+        app = FakeDaemonizedApp()
+        return app
+    
+    class DaemonizedApp(DaemonLite):
+        def run(self):
+            mainfunc()
+    app = DaemonizedApp(pidfile)
+    return app
+
+
 @begin.subcommand
 def start(reload: 'Make server autoreload (Dev only)'=False,
         debug_asyncio: 'Enable asyncio debug'=False,
-        cli : "enables an IPython CLI for debugging (not perational)"=False,
+        cli : "enables an IPython CLI for debugging (not operational)"=False,
         profile : "enables profiling / run stats (not operational)"=False,
+        # TODO: can be True if DaemonLite is a hard requirement
+        daemon : "if true pypeman will be started as daemon "=bool(DaemonLite),
+
         ):
-    """ Start pypeman """
-    reloader_opt(partial(main, debug_asyncio=debug_asyncio, cli=cli, profile=profile), reload, 2)
+    """ Start pypeman as daemon (or foreground process) """
+    main_func = partial(main, debug_asyncio=debug_asyncio, cli=cli, 
+            profile=profile)
+    start_func = partial(reloader_opt, main_func, reload, 2)
+
+    if reload:
+        print("RELOAD")
+        start_func()
+    else:
+        if daemon:
+            daemon = mk_daemon(main_func)
+            daemon.start()
+        else:
+            print("nodae")
```

Comment:
Remove print ? or make them homogeneous in Case.

Reply:
removed print